FirebaseInAppMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2020-02-18 -- v0.17.0
+- [added] Added support for data bundles for in-app messages - additional key-value pairs that can be sent along with an in-app message (#4922).
```

Comment:
The dash is a bit confusing since it implies in-app messages are additional ...

How about - Added support for data bundles for in-app messages. Data bundles are additional key-value pairs that can be sent along with an in-app message (#4922).

Reply:
Good point. Thanks for the suggestion, just changed it.